lib/embulk/input/marketo/lead.rb

```diff
@@ -91,7 +91,12 @@ def self.generate_columns(metadata)
 
         def init
           @columns = task[:columns]
-          @ranges = task[:ranges][index]
+          if preview?
+            # Try earlier date at first to reduce cache miss hit
+            @ranges = task[:ranges].flatten.sort_by{|range| Time.parse(range["to"])}.reverse
```

Comment:
Sorry, I'm a bit confused. Does "@ ranges" the last element at first, or this change intended to run newer at first?